core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -160,6 +162,22 @@ public void expandGlobalDefinitions() {
         }
     }
 
+    @JsonIgnore
+    public Function<Element, Set<Object>> getGroupByFunction() {
```

Comment:
The key should also include the Element identifiers (SOURCE, DESTINATION, DIRECTED, VERTEX) and the element group. These are all the things that make up the element key.